modules/private.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * Module Name: Private site
+ * Module Description: Make your site only visible to you and users you approve.
+ * Sort Order: 9
+ * First Introduced: ?
+ * Requires Connection: No
+ * Auto Activate: No
+ * Module Tags: Private
+ * Feature: Traffic
+ * Additional Search Queries: private, sandbox, launch, unlaunched
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Returns the private site template for private blogs
+ *
+ * @param object $wp Current WordPress environment instance (passed by reference).
+ */
+function privatize_blog( $wp ) {
```

Comment:
The functions in this file have fairly generic names. It would be best to either add the `jetpack_` prefix, or move everything into a `Jetpack_Private` class for example, to avoid conflicts with other plugins that may use functions with similar names.